### PR TITLE
Added profile view photo upload

### PR DESCRIPTION
**Pull request by anand-ts**
- Added photo upload in the ProfileView page

### CODE REVIEW

File: beauty-consulting-app/app/assets/components/ProfileImage.js [hunk 1]
```diff
@@ -56,22 +66,64 @@ const ProfileImage = ({ username }) => {
...
+			const dataUri = `data:${contentType};base64,${base64}`;
+
+			userContext.updateUserContext({
+				profilePhoto: dataUri,
+			});
+		} catch (error) {
+			// console.error("Error fetching updated profile picture:", error);
+		}
+	};
+
```
Comment (DavidHe137) [inline beauty-consulting-app/app/assets/components/ProfileImage.js:88]: This function seems to only be called after a photo is uploaded. This means that when I logout and login, I do not see the photo I uploaded previously. Maybe call this in a useEffect for the component?

File: beauty-consulting-app/app/Pages/ProfileView.js [hunk 2]
```diff
@@ -1,119 +1,437 @@
...
+			try {
+				const apiURL =
+					Constants.manifest?.extra?.apiUrl ||
+					process.env.EXPO_PUBLIC_API_URL;
+				if (!apiURL) {
+					console.error("apiURL not defined");
+					Alert.alert("Error", "API URL is not defined.");
+					setLoading(false);
+					return;
+				}
```
Comment (DavidHe137) [inline beauty-consulting-app/app/Pages/ProfileView.js:183]: This should not be necessary. See the refactored way to make http calls to the backend in #89.

File: beauty-consulting-app/app/Pages/ProfileView.js [hunk 3]
```diff
@@ -1,119 +1,437 @@
...
+	Modal,
+	TextInput,
+	Alert,
+	ActivityIndicator,
+	Image,
+} from "react-native";
 import { useNavigation } from "@react-navigation/native";
 import { UserContext } from "../contexts/userContext";
 import SignupBackground from "../assets/components/SignupBackground";
-import api from "utils/axios";
```
Comment (DavidHe137) [inline beauty-consulting-app/app/Pages/ProfileView.js:6]: I don't think this should be deleted as this is the new way we are making HTTP requests, as per #89.

File: beauty-consulting-app/app/Pages/ProfileView.js [hunk 4]
```diff
@@ -1,119 +1,437 @@
...
-			const res = api.delete(`/client/${userContext.username}`);
+			const apiURL =
+				Constants.manifest?.extra?.apiUrl ||
+				process.env.EXPO_PUBLIC_API_URL;
+			if (!apiURL) {
+				console.error("apiURL not defined");
+				Alert.alert("Error", "API URL is not defined.");
+				setLoading(false);
+				return;
+			}
```
Comment (DavidHe137) [inline beauty-consulting-app/app/Pages/ProfileView.js:143]: See #89, this should all not be necessary.

File: beauty-consulting-app/app/Pages/ProfileView.js [hunk 5]
```diff
@@ -1,119 +1,437 @@
...
+		try {
+			const apiURL =
+				Constants.manifest?.extra?.apiUrl ||
+				process.env.EXPO_PUBLIC_API_URL;
+			if (!apiURL) {
+				console.error("apiURL not defined");
+				Alert.alert("Error", "API URL is not defined.");
+				setLoading(false);
+				return;
 			}
```
Comment (DavidHe137) [inline beauty-consulting-app/app/Pages/ProfileView.js:95]: See #89, should not be needed.

**Review by DavidHe137 — CHANGES_REQUESTED** [review]
UI changes look pretty good. A little confused by the changes to the Schema, as well as some of the frontend API calls. 

Main issue is that after logging in, uploading a photo, logging out, and logging back in, I cannot see the photo I uploaded in my profile.

File: server/server.js
[the remark this answers is not in the document]
Comment (anand-ts) [reply server/server.js:1]: Yeah no changes, just added comments.

File: beauty-consulting-app/app/assets/components/ProfilePicture.js
[the remark this answers is not in the document]
Comment (anand-ts) [reply beauty-consulting-app/app/assets/components/ProfilePicture.js:1]: Temporarily commented the file to use the default profile-user.svg as a placeholder for profile picture at a later time.

File: server/model/account.js
[the remark this answers is not in the document]
Comment (anand-ts) [reply server/model/account.js:1]: I will double check the schema again and update my changes (possibly next sprint?)

File: beauty-consulting-app/app/Pages/ProfileView.js (hunk 3 above)
Comment (anand-ts) [reply beauty-consulting-app/app/Pages/ProfileView.js:6]: Added back.

File: beauty-consulting-app/app/Pages/ProfileView.js (hunk 5 above)
Comment (anand-ts) [reply beauty-consulting-app/app/Pages/ProfileView.js]: Resolved.

File: beauty-consulting-app/app/Pages/ProfileView.js (hunk 4 above)
Comment (anand-ts) [reply beauty-consulting-app/app/Pages/ProfileView.js]: Resolved.

File: beauty-consulting-app/app/Pages/ProfileView.js (hunk 2 above)
Comment (anand-ts) [reply beauty-consulting-app/app/Pages/ProfileView.js:115]: Resolved.

File: beauty-consulting-app/app/assets/components/ProfileImage.js (hunk 1 above)
Comment (anand-ts) [reply beauty-consulting-app/app/assets/components/ProfileImage.js:88]: Yeah, that's an issue as well. I will address this in the next sprint if that's okay.

File: beauty-consulting-app/app/Pages/ProfileView.js (hunk 2 above)
Comment (DavidHe137) [reply beauty-consulting-app/app/Pages/ProfileView.js:115]: This is still not quite right. All you should need is just a call to `api.get`

File: beauty-consulting-app/app/assets/components/ProfilePicture.js
Comment (DavidHe137) [inline beauty-consulting-app/app/assets/components/ProfilePicture.js:1]: Not sure what is going on with this file. If it is not used, and will not be used in the future, it should be deleted.

File: server/server.js
Comment (DavidHe137) [inline server/server.js:1]: As I mentioned in a previous comment, I don't really understand why this file has the changes that it does

File: beauty-consulting-app/app/Pages/ProfileView.js (hunk 2 above)
Comment (DavidHe137) [reply beauty-consulting-app/app/Pages/ProfileView.js:115]: Also I am still unable to see the profile photo after logging out. It seems like the photo is being stored properly in the database though.

**Review by DavidHe137 — CHANGES_REQUESTED** [review]
I still am unable to view the uploaded profilePhoto after logging out and logging in.

**Pull request closed without merging**